common/script/content/spells.js

```diff
@@ -0,0 +1,499 @@
+import t from './translation';
```

Comment:
Any reason moving the spells into their own content module shouldn't happen in `develop`?